waku/node/v2/rpc/wakucallsigs.nim

```diff
@@ -5,6 +5,7 @@ proc waku_publish(topic: string, message: seq[byte]): bool
 # TODO This should be properly done with rpc types, etc.
 proc waku_publish2(topic: string, message: seq[byte]): bool
 proc waku_subscribe(topic: string): bool
+proc waku_query(uuid: string, topics: seq[string]): bool
```

Comment:
Should the UUID be generated at the time of request perhaps?

Reply:
@oskarth for the time being I think its easier to do like this.

Reply:
ok